### PR TITLE
Created Blur Theme

### PR DESCRIPTION
**Pull request by HeyItsJono**
Created Blur theme for Themes addon.

### CODE REVIEW

File: Themes/blur/blur.css [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+    background-color: rgba(1,1,1,0);
+}
+
+.l-container.l-container--two-column .l-content, .l-container.l-container--two-column-dashboard .l-content {
+    background-color: rgba(1,1,1,0);
+}
+
+.post {
+	background: blue;
+	border-radius: 20px;
```
Comment (finagle29) [inline Themes/blur/blur.css]: `background: blue` makes the new post container a pretty harsh blue with the title, body, body and tags a white that sticks out.  The new post container also has white corners made visible by `border-radius` that can be fixed with
```css
.post-container {
    background: transparent;
}
```
image

File: Themes/blur/blur.css [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+    background-image: none;
+    background: rgba(0, 0, 0, 0);
+    background-color: rgba(0, 0, 0, 0);
+}
+
+.radar .radar_footer .radar_avatar:before {
+    background: rgba(0, 0, 0, 0);
+    background-color: rgba(0, 0, 0, 0);
+    background-image: none;
+}
```
Comment (finagle29) [inline Themes/blur/blur.css]: `background: rgba(0, 0, 0, 0)` is shorthand for `background-color: rgba(0, 0, 0, 0)` and sets all other `background-` prefixed attributes to their default values.  Lines 83 and 84 are not needed.

**Comment by finagle29** [comment]
There are a bunch of places where you're being redundant in the same manner as my line note on lines 82-84.  Fix those and the visual harshness encountered when making a new post and then change the body's image to point to `https://new-xkit.github.io/XKit/Themes/blur/body-background.jpg` (because that's where it'll be hosted after this PR is merged) and you're good to go!

**Comment by HeyItsJono** [comment]
Oops, didn't even notice the Post box thing, it completely slipped my mind to test that area! I've made the requested changes to the branch :) Also, just to clarify, the background image is a PNG file, did you guys want a JPG? I find PNG had a lot more dithering in the blur so it looked nicer, but I can go change it to JPG. As of right now the url is pointing at a PNG extension but I can change all that if you want.

**Comment by finagle29** [comment]
I don't think we have a preference for either file format, if you prefer a PNG that works!

`#content` is converted into `.l-content` in `themes.js` (by a little bit of code that I hope we can remove and just advise future theme developers that tumblr's not using that id anymore) so that style overridden by `.l-container.l-container--two-column .l-content {background-color: rgba(1,1,1,0);}`.  It's redundant code right now, but you might want something else to use that color or just remove it.

Other than the `#content` bit this looks great to me.

**Comment by finagle29** [comment]
Thanks for the contribution!
LGTM @homu r+

**Comment by HeyItsJono** [comment]
Thanks guys!

**Comment by ThePsionic** [comment]
Thank you too, @HeyItsJono :wink:

**Pull request merged**